src/routes/index.ts

```diff
@@ -0,0 +1,78 @@
+import express from 'express';
+import { UserManager } from '../../utils/generateUsers';
+import { UserRes } from '../types';
+import { schema, reduceErrorResponse } from '../../utils/validation';
+
+const router = express.Router();
+const userManager = new UserManager(5);
+
+router.get('/users', (req: express.Request, res: express.Response) => {
```

Comment:
it's better to create a separate file for user routes, f.e. `userRoutes`

Reply:
Fixed